powersimdata/network/tests/test_model.py

```diff
@@ -5,10 +5,10 @@
 
 def test_area_to_loadzone_argument_type():
     with pytest.raises(TypeError, match="area must be a str"):
-        area_to_loadzone("europe_tub", 3)
+        area_to_loadzone("usa_tamu", 3)
```

Comment:
Changing it into `usa_tamu` for faster test run?

Reply:
No. Previously we could instantiate the `ModelImmutables` class with `europe_tub` and obtain an incomplete mapping (see the `powersimdata.network.constants.europe` module), The other mappings, the one depending on the `Network` object (mostly loadzone related, e.g, `abv2loadzone` and so forth) were derived in the `TUB` class. Now we need the `zone` data frame to get all the model immutables and for `europe_tub` it comes from a `Network` object that we don't have in the tests. To check error raised, it is not worth creating a `TUB` object to access the `zone` data frame for the `europe_tub` model